packages/components/src/MultiSelect/MultiSelect.mdx

```diff
@@ -41,6 +41,52 @@ import { MultiSelect } from "@jobber/components/MultiSelect";
 
 <Props of={MultiSelect} />
 
+## Sizes
```

Comment:
These should be nested within the Usage Guidelines section (this "Sizes" header will become an H3, the "Small" and "Large" will become H4s) and have documentation on when to use each variation.

Reply:
Maybe something like

```suggestion
## Sizes

Adjusting the size of the `MultiSelect` can be used to help align with common adjacent elements such as InputText or Select.
```

And then you could even put both size examples into a single <Playground> to save vertical space.